common.py

```diff
@@ -9,262 +9,301 @@
 import discord
 from discord.ext import commands, tasks
 
+invites = {
+    # TTS bot
+    "513423712582762502": "https://discordapp.com/api/oauth2/authorize?client_id=513423712582762502&permissions=36719617&scope=bot",
+    # F@H
+    "565820959089754119": "https://discordapp.com/api/oauth2/authorize?client_id=565820959089754119&permissions=84992&scope=bot",
+    # Channel maker
+    "689564772512825363": "https://discordapp.com/api/oauth2/authorize?client_id=689564772512825363&permissions=68624&scope=bot",
+
+    # TTS bot dev for testing
+    "698218518335848538": "https://example.com"
+}
+
+colours = {
+    # TTS bot
+    "513423712582762502": 0x3498DB,
+    # F@H
+    "565820959089754119": 0xFE6215,
+    # Channel maker
+    "689564772512825363": 0x737F8D,
+    # TTS bot dev
+    "698218518335848538": 0x3498DB
+}
+
 
 def setup(bot):
-  bot.add_cog(Gnome(bot))
+    bot.add_cog(Gnome(bot))
+
 
 class Gnome(commands.Cog):
-  def __init__(self, bot):
-    self.bot = bot
-  def is_trusted(ctx):
-    if not os.path.exists("config.ini") and ctx.author.id in (341486397917626381, 438418848811581452): return True
-    elif os.path.exists("config.ini"): 
-      config = ConfigParser()
-      config.read("config.ini")
-      if str(ctx.author.id) in config["Main"]["trusted_ids"]: return True
-      
-    raise commands.errors.NotOwner
-  
-  #////////////////////////////////////////////////////////
-  @commands.command()
-  @commands.bot_has_permissions(send_messages=True, read_messages=True)
-  async def donate(self, ctx):
-    await ctx.send(f"To donate to support the development and hosting of {self.bot.user.mention}, you can donate via Patreon!\nhttps://www.patreon.com/Gnome_the_Bot_Maker")
-
-  @commands.command()
-  @commands.bot_has_permissions(send_messages=True, read_messages=True)
-  async def botstats(self, ctx):
-    message = cleandoc(f"""
+    def __init__(self, bot):
+        self.bot = bot
+
+    @staticmethod
```

Comment:
I'm not sure what `@staticmethod` does but Pycharm said to add it

I think it probably means that it doesn't use `self`